### PR TITLE
Task: Add universal header and footer

### PR DESCRIPTION
**Pull request by paulathevalley**
- Resolves https://github.com/FormidableLabs/radium-docs/issues/28
- Upgrade `formidable-landers` to use new header and footer component
- Upgrade `builder-docs-archetype` and `react` to fix the multiple copies of react installed error 
- Refactor the sidebar toggle on resize; the main content was getting cut off by moving too far to the left... I think this works.

@ebrillhart Do the links (Docs, Issues, View Source on GitHub) look good to you? Is there anything missing?

/cc @ebrillhart @coopy @bmathews @kylecesmat 

screencapture-localhost-3000-1473722442192
screen shot 2016-09-12 at 4 36 16 pm
screen shot 2016-09-12 at 4 36 26 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,23 +20,24 @@
...
     "history": "~1.17.0",
     "markdown-it": "^7.0.0",
     "markdown-it-named-headers": "0.0.4",
     "prismjs": "^1.5.1",
     "radium": "*",
-    "react": "15.2.1",
-    "react-dom": "15.2.1",
+    "react": "^15.0.1",
+    "react-addons-test-utils": "^15.0.1",
+    "react-dom": "^15.0.1",
```
Comment (bmathews) [inline package.json]: downgrading react?

Comment (bmathews) [inline package.json]: nevermind, it's my first time using computers

**Comment by ebrillhart** [comment]
LGTM! we'll need an about link eventually but obviously that would be v confusing right now as it would link to nothing

File: package.json (hunk 1 above)
Comment (paulathevalley) [inline package.json]: 😬 mine too. I matched it to the react dependency in `radium`, so they’re both matchy-matchy.

**Comment by bmathews** [comment]
:shipit: !

**Pull request merged**